knowledge_graph/make_graph.py

```diff
@@ -345,30 +190,26 @@ def makeGraph(onto_path, edge_path, output_folder_path):
     # print(list(default_world.inconsistent_classes()))
 
     # Read in the triples data
-    df = pd.read_csv(edge_path)
+    ## DMARX - csv via make_network.outputEdges()
+    #          via node_network.result
+    # ... If we've already processed the ontology through the Network object,
+    # why do we need to reload it here?
+    # Can we move add_ontology_data_to_graph_nodes to network_class?
+    df_edges = pd.read_csv(edge_path)
 
-    G = nx.DiGraph()  # There should not be duplicate edges that go the same direction.
-    # If so, need to throw an error.
+    G = nx.DiGraph()
+    for src, tgt, kind in df_edges.values:
+        G.add_edge(src, tgt, type=kind, properties=None)
 
-    edges = convert_dataframe_to_edges(df)
-    add_edges_to_graph(edges, G)
     add_ontology_data_to_graph_nodes(G, onto)
     to_remove = set_edge_properties(G)
     remove_edge_properties_from_nodes(G, to_remove)
 
     save_graph_to_pickle(G, output_folder_path)
-    # save_graph_to_gexf(G, output_folder_path)
-    # save_graph_to_gml(G, output_folder_path)
-    # save_graph_to_graphml(G, output_folder_path)
-    # save_graph_to_yaml(G, output_folder_path)
-    # save_graph_to_json(G, output_folder_path)
 
     valid_test_ont = get_valid_test_ont()
```

Comment:
these lines can't work if you deleted these functions, right?

Reply:
As I mentioned above, they were moved to another file from which they are imported at the top of make_graph.py